poetry_scripts.py

```diff
@@ -90,7 +90,7 @@ def update_client():
     check_exit_code(
         "openapi-python-client update\
             --url https://raw.githubusercontent.com/fishjam-dev/"
-        "fishjam/main/openapi.yaml \
+        "fishjam/feat/cluster-health-report/openapi.yaml \
             --config openapi-python-client-config.yaml \
```

Comment:
We should keep using `main` here and only generate the SDKs after the relevant PR in `fishjam` gets merged